commune_app/tests/test_commune.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from commune_app.all_models.communes import Commune
+
+
+NAME = "test_commune"
+DESCRIPTION = "this is a description for test coomune"
+
+NAME1 = "test_commune1"
+DESCRIPTION1 = "description1"
+
+USERNAME_IN_DB = "testUser123"
+
+
+@pytest.fixture
+def commune0():
+    return Commune(name=NAME, description=DESCRIPTION)
+
+
+@pytest.fixture
+@pytest.mark.django_db()
+def my_commune0(commune0):
+    commune0.save()
+    return commune0
```

Comment:
maybe it will be better to create it in one fixture like:
```suggestion
@pytest.fixture
@pytest.mark.django_db()
def my_commune0():
    commune0 = Commune(name=NAME, description=DESCRIPTION)
    commune0.save()
    return commune0
```